esp/src/src/Utility.ts

```diff
@@ -23,6 +23,12 @@ export function xmlEncode2(str) {
         ;
 }
 
+export function decodeHtml(html) {
```

Comment:
Wouldn't this be better titled "decodeTextAreaHtml"?

Reply:
@miguelvazq not really its a general purpose HTML -> Text decoder which happens to use a textarea (internally) to do the heavy lifting.